misc/mzutil/await_view_states.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""await_view_states
+
+Script to query all materialized views in a Materialize database and wait until each view matches
+the exact output as captured in the snapshot files.
+
+Prints timing information to indicate how long each view took to reach the desired state.
+"""
+
+import argparse
+import glob
+import io
+import os
+import time
+
+import psycopg2
+import psycopg2.errors
+
+
+def view_names(conn):
+    """Return a generator containing all view names in Materialize."""
+    with conn.cursor() as cursor:
+        cursor.execute("SHOW VIEWS")
+        for row in cursor:
+            yield row[0]
```

Comment:
Doing this correctly should immediately cause bin/pycheck to fail because of lack of explicit types, if it doesn't that means we've got something wrong with our config somewhere.